assets/lang/sklmessages_en_US.properties

```diff
@@ -62,18 +61,16 @@ editor.close=Close launcher when game starts
 editor.keep=Keep the launcher open
 editor.snap=Snapshots
 editor.snap.tooltip=Allow use of experimental development versions ("snapshots")
-editor.beta=Beta
-editor.beta.tooltip=Allow use of "Beta" Minecraft versions (From 2010–2011)
-editor.alpha=Alpha
-editor.alpha.tooltip=Allow use of "Alpha" Minecraft versions (From 2010)
+editor.historical=Historical
+editor.beta.tooltip=<todo>
```

Comment:
This two options `editor.beta` and `editor.alpha` will no longer be split just like in Mojang's Launcher